mongodbatlas/data_source_mongodbatlas_search_analyzers.go

```diff
@@ -0,0 +1,115 @@
+package mongodbatlas
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+	"net/http"
+)
+
+func dataSourceMongoDBAtlasSearchAnalyzers() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceMongoDBAtlasSearchAnalyzersRead,
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"cluster_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"page_num": {
+				Type:     schema.TypeInt,
+				Optional: true,
+			},
+			"items_per_page": {
+				Type:     schema.TypeInt,
+				Optional: true,
+			},
+			"results": {
+				Type:     schema.TypeSet,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"base_analyzer": {
+							Type:     schema.TypeString,
+							Computed: true,
+							Required: false,
+						},
+						"ignore_case": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"max_token_length": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Optional: false,
+						},
+						"stem_exclusion_set": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+						"stopwords": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+					},
+				},
+			},
+			"total_count": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func dataSourceMongoDBAtlasSearchAnalyzersRead(d *schema.ResourceData, meta interface{}) error {
+	// Get client connection.
+	conn := meta.(*MongoDBClient).Atlas
+	projectID, projectIDOK := d.GetOk("project_id")
+	clusterName, clusterNameOK := d.GetOk("cluster_name")
+
+	if !projectIDOK || !clusterNameOK {
```

Comment:
Isn't this the same as `!(projectIDOK || clusterNameOK )`

Reply:
the `project_id` is the same as `org_id` but not the `cluster_name`

Reply:
project_id is NOT the same as org_id.   Org_id is the top of the hierarchy, an org has one more more projects.

Reply:
@themantissa according to the atlas documentation (https://docs.atlas.mongodb.com/reference/api/fts-indexes-create-one/) it says

> 
Groups and projects are synonymous terms. Your {GROUP-ID} is the same as your project ID. For existing groups, your group/project ID remains the same. The resource and corresponding endpoints use the term groups.

Reply:
@themantissa ohhhhh god! I saw my error

I was confused about `org_id` and `group_id`

Reply:
Yup, you caught it!  groups and projects are synonymous but not orgs and groups/projects :)

Reply:
thanks! and yes I was refering to `project_id = group_id`